SIL.WritingSystems/Sldr.cs

```diff
@@ -409,49 +410,90 @@ private static void LoadLanguageTagsIfNecessary()
 				CreateSldrCacheDirectory();
 
 				cachedAllTagsPath = Path.Combine(SldrCachePath, "langtags.json");
+				string etagPath;
+				etagPath = Path.Combine(SldrCachePath, "langtags.json.etag");
 				var sinceTime = _embeddedAllTagsTime.ToUniversalTime();
 				if (File.Exists(cachedAllTagsPath))
 				{
 					var fileTime = File.GetLastWriteTimeUtc(cachedAllTagsPath);
 					if (sinceTime > fileTime)
+					{
 						// delete the old langtags.json file if a newer embedded one is available.
 						// this can happen if the application is upgraded to use a newer version of SIL.WritingSystems
 						// that has an updated embedded langtags.json file.
 						File.Delete(cachedAllTagsPath);
+						File.Delete(etagPath);
+					}
 					else
 						sinceTime = fileTime;
 				}
 				sinceTime += TimeSpan.FromSeconds(1);
```

Comment:
Since we don't send `sinceTime` to the server, we no longer need that variable here - we only need it above in line 419 so that we can compare it to `fileTime`.